### ISSUE TITLE
Certain errors during create/init are outputted only in STOUT not STDERR

### ISSUE THREAD

**Issue by razine-bensari** [edgelesssys/constellation#444]
### Issue description
When running certain commands that results in an error, such as using the wrong credentials in constellation-conf.yaml or a CSP credentials with not enough privileges.

### To reproduce

1. Run `constellation config generate`
2. Add your credentials in constellation-conf.yaml file (maybe falsify your credentials to trigger an error when exeucting other commands after)
3. Run `constellation create --control-plane-nodes 1 --worker-nodes 1 --name test -y` to try to create a cluster
4. Notice how there is an error in the STDOUT but not in STDERR. (You may need to monitor both STDOUT and STDERR beforehand to properly replicate the bug)
5. This is a sample of the error that could be shown:
```
An error occurred: PUT https://management.azure.com/subscriptions/6b6a1f27-55c1-4b1d-969b-60a3c9eebe64/resourceGroups/azure-customertwo-az-sandbox-84v4y/providers/Microsoft.Insights/components/constellation-insights-nm4rV
--------------------------------------------------------------------------------
RESPONSE 403: 403 Forbidden
ERROR CODE: AuthorizationFailed
--------------------------------------------------------------------------------
{
  "error": {
    "code": "AuthorizationFailed",
    "message": "The client '<ID>' with object id '<ID>' does not have authorization to perform action 'Microsoft.Insights/components/write' over scope '/subscriptions/<SUBSCRIPTION_ID>/resourceGroups/<RESOURCE_GROUP_NAME>/providers/Microsoft.Insights/components/constellation-insights-nm4rV' or the scope is invalid. If access was recently granted, please refresh your credentials."
  }
}
--------------------------------------------------------------------------------

Attempting to roll back.
```

### Environment

- `constellation version`:
- `constellation-conf.yaml`
  - (make sure to remove sensitive information, e.g., `yq e 'del(.provider.*.project)' constellation-conf.yaml`)
- VM type used to run Constellation.

### Expected behavior
As a developer, I expect a failure to output both on STDOUT and STDERR so I can integrate and handle failures more easily

**Comment by datosh** [comment]
Thanks for reporting this issue, @razine-bensari, and I agree with the expected behavior.
We will schedule this to be included in v2.3 release!

**Comment by datosh** [comment]
hey @razine-bensari 
I submitted a PR that brings more consistency in that errors and warnings are reported to `stderr`.

I don't see a benefit in duplicating this information by writing them to `stdout` and `stderr`. If this is required the client should be able to add those two streams back together.

Let me know if this solves your issues!
I was not able to reproduce the specific Azure error. If you are able to provide me with detailed instructions I'm happy to test this or fix other paths I might have overlooked.
